README.md

```diff
@@ -152,9 +152,13 @@ If the script is connecting to any local apps/services you need to forward the p
 container. The easiest way to do this is to use `--net=host` argument for `docker run`:
 
 ```
-docker run --net=host digitalmarketplace/scripts scripts/index-services.py dev ...
+docker run --net=host digitalmarketplace/scripts scripts/index-to-search-service.py dev ...
 ```
 
+**Note that this will use the latest image built from master, and therefore (at present) all
```

Comment:
I don't think this note should live under "Running scripts with Docker" - this seems like more a description of how the repository is used, rather than anything particularly related to Docker (Jenkins jobs that run scripts outside of Docker also pull the current master branch each time).  It could be a small introductory paragraph about the repo - "A collection of scripts, some of which are run on a schedule on Jenkins.  Jenkins always runs current master so changes will take immediate effect".

I'm not sure it needs all the bold.